controller/src/beerocks/cli/beerocks_cli_bml.cpp

```diff
@@ -1953,9 +1953,9 @@ int cli_bml::start_dcs_single_scan(const std::string &radio_mac, int32_t dwell_t
     std::cout << "start_dcs_single_scan, mac=" << radio_mac << ", dwell_time=" << dwell_time
               << ", channel_pool=" << channel_pool << std::endl;
 
-    if (dwell_time <= 0) {
-        std::cout << __func__ << ", invalid input: dwell_time(" << dwell_time << ") <= 0"
-                  << std::endl;
+    if (dwell_time < 0) {
+        std::cout << __func__ << ", invalid input: dwell_time(" << dwell_time
+                  << ") < 0. Only positive values are supported!" << std::endl;
```

Comment:
This commit (67a5cb412e2f938b4334e7fc5fd9ed6dba300c10: beerocks_cli_bml: start_dcs_single_scan: remove dwell time 0 varifica…) headline is too long and exeeds 72 characters.

Also, the print would look like this:
"invalid input: dwell_time(-3) < 0. Only positive values are supported!"

I think it would look better like this:
"invalid input, dwell_time='-3', Only positive values are supported!
(nitpick)

Reply:
I agree it looks better without the `<0`
Changed